kong-0.11.2-0.rockspec

```diff
@@ -24,7 +24,7 @@ dependencies = {
   "luatz == 0.3",
   "lua_system_constants == 0.1.2",
   "lua-resty-iputils == 0.3.0",
-  "luacrypto == 0.3.2",
+  "luaossl == 20171028",
```

Comment:
@daurnimator Would you mind adding the `CRYPTO_DIR` variable in the `install` rule of the Makefile as part of the same commit? It seems luaossl requires both `OPENSSL_DIR` _and_ `CRYPTO_DIR` (previously, we only needed `OPENSSL_DIR`. It can probably just be:
```
@luarocks make OPENSSL_DIR=$(OPENSSL_DIR) CRYPTO_DIR=$(OPENSSL_DIR)
```